assets/src/edit-story/components/tablist/styles.js

```diff
@@ -0,0 +1,163 @@
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_VARIANTS,
+  Headline,
+  ThemeGlobals,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../design-system';
+
+export const Tablist = styled.div`
+  width: 308px;
+  background: ${({ theme }) => theme.colors.bg.primary};
+`;
+Tablist.propTypes = {
+  // label to describe purpose of tabs
+  'aria-label': PropTypes.string.isRequired,
+};
+
+export const PanelWrapper = styled.div`
+  width: 100%;
+  background: ${({ theme }) => theme.colors.bg.primary};
+
+  ${({ isExpanded }) =>
+    isExpanded &&
+    css`
+      & > ${TabButton} {
+        ${SmallHeadline} {
+          opacity: 1;
+        }
+
+        :hover,
+        :focus {
+          background-color: theme.colors.bg.primary;
+        }
+
+        svg {
+          transform: rotate(0);
+        }
+      }
+
+      & > ${TabPanel} {
+        height: 560px;
+        padding: 16px;
+        visibility: visible;
+      }
+    `};
+`;
+
+export const SmallHeadline = styled(Headline).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XX_SMALL,
+})``;
+
+export const TabButton = styled(Button).attrs({
+  variant: BUTTON_VARIANTS.PLAIN,
+})`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  width: 100%;
+  padding: 16px;
+  margin-bottom: 1px;
+  border-radius: 0;
+  box-shadow: 0px -1px 0 0 ${({ theme }) => theme.colors.divider.tertiary};
+
+  &,
+  :hover,
+  :focus,
+  .${ThemeGlobals.FOCUS_VISIBLE_SELECTOR} {
+    background-color: ${({ theme }) => theme.colors.bg.secondary};
+  }
+
+  :focus,
+  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR} {
+    z-index: 2;
+  }
+
+  ${SmallHeadline} {
+    opacity: 0.8;
+  }
+
+  ${({ status }) =>
+    status === 'danger' &&
+    css`
+      &,
+      ${SmallHeadline} {
+        color: ${({ theme }) => theme.colors.fg.negative};
+      }
+
+      ${Badge} {
+        background-color: ${({ theme }) => theme.colors.fg.negative};
+      }
+    `};
+`;
+
+export const ButtonText = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: -12px;
+`;
+
+export const IconContainer = styled.div`
+  height: 32px;
+  width: 32px;
+
+  color: ${({ theme }) => theme.colors.fg.primary};
+
+  svg {
+    transform: rotate(-90deg);
+    transition: transform 300ms ease-in-out;
+  }
+`;
+
+export const Badge = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  width: 20px;
+  border-radius: ${({ theme }) => theme.borders.radius.round};
+  background-color: ${({ theme }) => theme.colors.fg.primary};
+
+  ${SmallHeadline} {
+    color: ${({ theme }) => theme.colors.inverted.fg.primary};
+    line-height: 20px;
+  }
+`;
+
+export const TabPanel = styled.div`
+  height: 0;
+  padding: 0;
+  visibility: hidden;
+  overflow-x: scroll;
```

Comment:
what was the need for this style here? should this be overflow-y?

Reply:
🤦